Cargo.toml

```diff
@@ -105,9 +103,9 @@ members = [
 	"primitives/keyring",
 	"primitives/offchain",
 	"primitives/panic-handler",
-	"primitives/peerset",
 	"primitives/phragmen",
 	"primitives/rpc",
+	"primitives/rpc/api",
```

Comment:
I don't feel like `rpc/api` definitions fit the `primitives` directory. It's just a set of API that are exposed by the default substrate node. As a side effect it also generates a bunch of `rpc-client` structs, though that seem to be required for some `paint` module. I wonder if the rpc client doesn't deserve a separate place (maybe `util`?).

Perhaps we try too much to obey the rules in the script without actually looking deeper what we try to achieve. Seems to me that `primitives` is now just becoming a `common` stuff between runtime and client, instead of being a merely a set of primitive types/structs.

Reply:
agreed, this is wrong. moved it back to client, moved the only import out of palette into utils